SU-HKKU/cardinal_service_profile
# CSD-473 Added opportunity program, updated twig template and config

# READY FOR REVIEW

# Summary
- Added opportunity program, updated twig template and config

# Needed By (Date)
- 09/11

# Urgency
- medium

# Steps to Test

1. Pull this PR
2. Clear cache
3. Check for:
- Opportunity Program should appear on opportunity list page.
- Comma and `in` should only appear if the neighboring tags are selected.

# Associated Issues and/or People
- [CSD-473](https://stanfordits.atlassian.net/browse/CSD-473)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
